packages/sfpowerscripts-cli/src/impl/parallelBuilder/BuildImpl.ts

```diff
@@ -545,13 +540,13 @@ export default class BuildImpl {
       }
     } else {
       if (packageType === "Source" || packageType == "Unlocked") {
-        result = this.createSourcePackage(
+        result = await this.createSourcePackage(
```

Comment:
I don't think we should await here, it should return the promise to make it parallel

Reply:
The invocation of the createPackage() method by the scheduler is asynchronous, but the logic within the method should be synchronous.